### ISSUE TITLE
Kicad simulator is broken due to /app/lib/libngspice.so not being found

### ISSUE THREAD

**Issue by jmaibaum** [flathub/org.kicad_pcb.KiCad#29]
The Kicad simulator in Eeschema (Tools -> Simulator) is not working, because /app/lib/libngspice.so is not found. When trying to start the simulator, KiCad displays an error message and then quits due to an unhandled exception.

#25 has a fix for this by adding `/app/lib` to `LD_LIBRARY_PATH` env var in the sandbox during finishing. With this fix applied, the Kicad simulator runs as expected.

**Comment by jmaibaum** [comment]
Just in case people are in search for a workaround, the following `flatpak override` command adds the change from this pull request to your local installation:

```bash
flatpak override --user --env=LD_LIBRARY_PATH=/app/lib org.kicad_pcb.KiCad
```

To any flathub maintainers reading this: I am no expert, so what I am suggesting with this PR might better be fixed upstream, if it is intended that `/app/lib` is not being included in `LD_LIBRARY_PATH` at runtime. Any guidance as to what should be done (and where, i.e. in the flatpak manifest or upstream) to fix this issue for the humble user of the software properly is highly appreciated.

The `flatpak override` is a workaround for the user who simply wants to work with the KiCad ngspice simulator, though.

**Comment by jmaibaum** [comment]
Closed by #27
